data/profile.d/Makefile.am

```diff
@@ -24,6 +24,7 @@ dist_config_DATA    = \
 	fedora-iot.conf \
 	fedora-kde.conf \
 	fedora-kinoite.conf \
+	fedora-sericea.conf \
```

Comment:
Can we just wildcard these?

Reply:
IIRC it is not recommended and we had a PR from outside to change place we had back than.

See here: https://github.com/rhinstaller/anaconda/pull/3619/commits/5c7ea9c2cee989d0b1b03cc7cfaf24f951271920

Reply:
Ah, sorry then - I thought that's only in srpms.